### PR TITLE
Milestone four

### PR DESCRIPTION
**Pull request by Addis-Belete**
This milestone added a feature to filter a book by category on book-store

### CODE REVIEW

File: src/components/CategoryFilter.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
```
Comment (bellom) [inline src/components/CategoryFilter.js]: With the ESLint inline comment, you are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.

If you are using VSCode, you can delete the inline comment and hover over the error(configure Eslint checks locally). And you will see your options. Click on the more info button that will take you to a GitHub repo where it is explained why you are getting this error. I find this feature very educational. Please, try to learn and implement the necessary rules so you don't have to keep this disabling line on top of each file. I believe you can do this! 💪😇.

File: src/containers/BooksList.js [hunk 2]
```diff
@@ -1,30 +1,47 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-unused-expressions */
+/* eslint-disable array-callback-return */
 /* eslint-disable react/forbid-prop-types */
```
Comment (bellom) [inline src/containers/BooksList.js]: With the ESLint inline comment, you are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.

File: src/reducers/filter.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-expressions */
```
Comment (bellom) [inline src/reducers/filter.js]: With the ESLint inline comment, you are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.

File: src/reducers/index.js [hunk 4]
```diff
@@ -1,9 +1,11 @@
 /* eslint-disable import/extensions */
```
Comment (bellom) [inline src/reducers/index.js]: With the ESLint inline comment, you are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @Addis0943  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check my comments under this review;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best**

**Comment by Addis-Belete** [comment]
Hello TSE
Thank you for your feedback. I did on all requested changes. kindly review it.
Regards

**Review by uche-inyama — APPROVED** [review]
## Approved :heavy_check_mark: 

### Hi Addis :slightly_smiling_face: 

We are good here, move on with the next milestone.

### Happy coding :man_technologist: 

#### ``` Uchechukwu Henry Inyama ```

**Pull request merged**